tests/profiler/test_profiler.py

```diff
@@ -258,12 +258,6 @@ def test_pytorch_profiler_describe(pytorch_profiler):
     assert len(data) > 0
 
 
-def test_pytorch_profiler_raises(pytorch_profiler):
-    """Ensure errors are raised where expected."""
-    with pytest.raises(MisconfigurationException, match="profiled_functions` and `PyTorchProfiler.record"):
-        PyTorchProfiler(profiled_functions=["a"], record_functions=["b"])
-
-
```

Comment:
I wasn't entirely sure if removing this test entirely was the right call. After removing `profiled_functions` this raises a `TypeError` instead since `record_functions` expects a `Set` not a `List`. I could instead just update the test to check for that raised `TypeError` if desired.

Reply:
All good !